front_end/server/lib/routes/servers.js

```diff
@@ -1,4 +1,19 @@
 const express = require('express');
 const router = express.Router();
 const Server = require('../models/server');
-const bodyParser = require('body-parser').json();
\ No newline at end of file
+const bodyParser = require('body-parser').json();
+
+router
+  .get('/', (req, res, next) => {
+    Server.find()
+      .lean()
```

Comment:
Lean is great! It's a mongoose method that returns regular javascript objects from your database query instead of full-featured Mongoose documents. In this case, we won't need all that extra jazz (getters, setters, save, etc), so lean() gets us better performance.